lib/fluent/plugin/out_kafka2.rb

```diff
@@ -36,6 +36,9 @@ class Fluent::Kafka2Output < Output
 
     config_param :get_kafka_client_log, :bool, :default => false
 
+    config_param :ignore_exceptions, :string, :default => nil, :desc => "Ignore exception list split by comma (,)"
```

Comment:
use `array` type, not `string`: https://docs.fluentd.org/v1.0/articles/api-config-types#:array